File: Client/Client_test_AxelRivera/src/app/models/reporte-peticiondto.ts
export interface ReportePeticionDto{
  codigoCliente:number,
  nombreCliente:string,
  direccionCliente:string,
  telefonoCliente:string,
  idFactura:number,
  reportePeticionDetalleDto: Array<ReportePeticionDetalleDto>
}

export interface ReportePeticionDetalleDto{
  nombreProducto:number,
  cantidadVendida:number,
  precioUnitario:number,
}

File: WebApi.test_AxelRivera/test_AxelRivera.WebApi/Features/Reporte/Dto/ReportePeticionDto.cs
namespace test_AxelRivera.WebApi.Features.Reporte.Dto
{
    public class ReportePeticionDto
    {
        public int codigoCliente { get; set; }

        public string nombreCliente { get; set; } = null!;

        public string direccionCliente { get; set; } = null!;

        public string telefonoCliente { get; set; } = null!;

        public int idFactura { get; set; }

        public DateOnly fechaFactura { get; set; }

        public List<ReportePeticionDetalleDto> reportePeticionDetalleDto { get; set; }
    }
}
